Samples/iOS-Swift/iOS-SwiftUITests/SDKPerformanceBenchmarkTests.swift

```diff
@@ -0,0 +1,73 @@
+import XCTest
+
+class SDKPerformanceBenchmarkTests: XCTestCase {
+    func testCPUBenchmark() throws {
+        try XCTSkipUnless(UIDevice.current.systemVersion.components(separatedBy: ".").first ?? "" == "15", "Only run benchmarks on iOS 15.")
```

Comment:
`l`: Why only on iOS 15?

Reply:
After discussing the issue with iOS 10, we decided we can start by only running the benchmark on latest iOS. If we want to look for variations later due to API differences we can make changes here.